CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Notable changes.
 
 ## August 2022
 
```

Comment:
I'd love for us to get this small change out before releasing today

https://github.com/devcontainers/cli/pull/141